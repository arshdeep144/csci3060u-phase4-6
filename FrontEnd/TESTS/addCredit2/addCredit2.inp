login
admin02
addcredit
user12
1001.00
logout