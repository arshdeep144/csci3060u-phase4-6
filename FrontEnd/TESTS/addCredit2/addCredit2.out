Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter a username
Please enter how much credit you would like to add
Maximum credit transfer exceeded
Please enter how much credit you would like to add
Adding credits...
Subtracting credits...
New credits for Admin are: 9999 and new credits for user are: 0
Welcome to the main menu. Please enter a command to continue: 
Command not Recognized
